docs/project/board/asahi-board.md

```diff
@@ -0,0 +1,21 @@
+---
+title: Asahi Linux Board
+---
+
+The Asahi Linux Board is the governing body of the Asahi Linux project. The board
+handles the administration of the project, including day-to-day operations, finances,
+and the Code of Conduct.
+
+The board meets once a month.
+
+## Current Members
+- Alyssa Rosenzweig
+- Neal Gompa
+- Davide Cavalca
+- Janne Grunau
+- Sven Peter
+- chaos_princess
+- James Calligeros
+
+## Meeting Minutes (times in US Eastern Standard Time)
```

Comment:
Just say `US Eastern Time` because otherwise this is going to be wrong throughout the year between Standard and Daylight Saving Time.

Reply:
Is there anything wrong with just standardising on an agreed upon timezone? This is why I used UTC initially. It doesn't matter what clock time the meeting was held at for any particular person in any particular location.

Reply:
The calendar is set to `US/Eastern`, so we should use that.

Reply:
I still don't think that matters at all for the purposes of keeping minutes but sure